src/containers/FormikForm/FormikCompetencesContent.tsx

```diff
@@ -132,7 +111,11 @@ const FormikCompetencesContent = ({
       {competences.map((competence, index) => (
         <FormPill
           id={index.toString()}
-          label={competence.code + ' - ' + competence.title}
+          label={`${competence.code} - ${
+            competence.title
+              ? competence.title
+              : t('form.competences.titleNotFound')
```

Comment:
Nokke sånt er meir konsist.
```suggestion
            competence.title || t('form.competences.titleNotFound')
```